src/vtt-segment-loader.js

```diff
@@ -1023,9 +1028,20 @@ export default class VTTSegmentLoader extends videojs.EventTarget {
   }
 
   parseVTTCues_(segmentInfo) {
+    let decoder;
+    let convertData = false;
```

Comment:
Well, if this variable is `true` it means we have to decode both the map and segment data, not just the map. how does `decodeBytesToString` sound to you